tools/setupResourceScripts/suites/datarepo_azure_omop_datasets.json

```diff
@@ -8,7 +8,15 @@
       "stewards": ["DataRepoAdmins@dev.test.firecloud.org"],
       "custodians": [],
       "snapshot_creators": [],
-      "snapshots": [],
+      "snapshots": [
+        {
+          "name": "Azure_OMOP_Snapshot",
+          "description": "Test Azure Snapshot with OMOP data",
+          "readers": [
+            "JadeStewards-dev@dev.test.firecloud.org"
```

Comment:
Ditto for the below script, but Doug just merged a PR which paves the way for the removal of these stewards groups. Is there another reader setting that makes sense?

Reply:
I think we could continue to use the stewards group in this capacity. I believe Doug's PR  eliminates using the stewards group by default in regular app operations. We don't necessarily need to delete the underlying terra group. 
This is a use case particular to test resources. The idea is that if test resources are manually created, we can automatically give access to our team available for manual click testing. 
To avoid confusion, I've created a new terra group for this use, https://bvdp-saturn-dev.appspot.com/#groups/DataRepoTestResourceAccess, that is a little more trimmed down and a more specific name. Alternatively, we don't need to automatically add the group of users on test creation. (except for test resources created for use with TDR UI e2e tests)

Reply:
Ah, so that's true about what Doug's PR does, but it was the first step of the following proposal:

https://docs.google.com/document/d/1PV93hNCEEILZDvXJmFpUCuHuiVcnRrfEVd-uJ1eYQ5U/edit#heading=h.fgeyidbrjlh2

1. Stop granting access to the stewards group for all new dataset google projects in TDR code.
2. Grant the service account hca-dagster-runner@mystical-slate-284720.iam.gserviceaccount.com the ability to set snapshot to public
**3. Remove all members of the stewards group and see who screams**
4. (More thorough excising) 

> To avoid confusion, I've created a new terra group for this use… that is a little more trimmed down and a more specific group name
Thanks for switching to a new group more tailored to the use!  I think that's good enough for now and we can revisit in the future.  (What it feels like these resources want to be are public in some way, but publicizing resources currently have technical constraints associated with them.)

>(except for test resources created for use with TDR UI e2e tests)
We should begin thinking about what running TDR UI e2e tests off of BEEs would look like in this regard, since BEE-ification of tests is likely to be intertwined with modernizing TDR's GHAs and could come sooner than we originally thought.

Reply:
RE: Bees - Yeah, that's a good point. I'm really not sure what the UI E2E tests would look like on a BEE because we do require a good bit of preseeding. We could probably still add the test uesrs to a group, though and add them to the preseeded resources.